deployment/bare_metal/.env.dist

```diff
@@ -118,4 +118,13 @@ NGINX_SSL_INCLUDE=/etc/letsencrypt/options-ssl-nginx.conf
 # LEGACY
 NGINX_REWRITE_LEGACY_URLS=false
 DEFAULT_PUBLISHER_ID=2896
-WEBHOOK_ELOPAGE_SECRET=secret
\ No newline at end of file
+WEBHOOK_ELOPAGE_SECRET=secret
+
+WALLET_AUTH_URL=https://stage1.gradido.net/authenticate?token={token}
+WALLET_URL=https://stage1.gradido.net/login
```

Comment:
Why did you add these lines again? They are now further up and end in path and no longer in url.
```suggestion
```